redash/destinations/hangoutschat.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import requests
+
+from redash.destinations import *
+from redash.utils import json_dumps
+
+
+class HangoutsChat(BaseDestination):
```

Comment:
```suggestion
class HangoutsChat(BaseDestination):
    @classmethod
    def name(cls):
        return "Google Hangouts Chat"
    
    @classmethod
    def type(cls):
        return "hangouts_chat"
```

Small change to show a friendlier name and split the two words in the type identifier.